core/modules/order/store/actions.ts

```diff
@@ -45,26 +47,48 @@ const actions: ActionTree<OrderState, RootState> = {
             headers: { 'Content-Type': 'application/json' },
             mode: 'cors',
             body: JSON.stringify(order)
-          },
+          }
         })
         Vue.prototype.$bus.$emit('notification-progress-stop')
-        if (task.resultCode !== 500) {
+
+        if (task.resultCode === 200) {
           order.transmited = true
           commit(types.ORDER_PLACE_ORDER, order) // archive this order but not trasmit it second time
           commit(types.ORDER_LAST_ORDER_WITH_CONFIRMATION, { order: order, confirmation: task.result })
           Vue.prototype.$bus.$emit('order-after-placed', { order: order, confirmation: task.result })
+
+          return task
```

Comment:
Aren’t we showing which fields do containa errors  as Youu initially planned?

Reply:
@pkarw no, it will be added in separate PR

Reply:
Ok